src/rust/ide/view/graph-editor/src/component/breadcrumbs/project_name.rs

```diff
@@ -315,20 +317,16 @@ impl ProjectName {
 
             // === Selection ===
 
-            select <- any(&frp.select,&frp.input.start_editing);
```

Comment:
Earlier the `frp.input.start_editing` input was used here. It is not used anymore. Could you explain more why it was not necessary? I'm just trying to understand this change and why actually it solves the problem - any explanation why it solves it would be great

Reply:
The actual fix is the removal of `text.set_focus(true);` on `select`. The select should not do anything to the text field but change its colour, as its only meant to be a highlight. So, `start_editing` should also not change the `select` status.